packages/discord.js/src/structures/Presence.js

```diff
@@ -254,7 +254,8 @@ class Activity {
         this.type === activity.type &&
         this.url === activity.url &&
         this.state === activity.state &&
-        this.details === activity.details)
+        this.details === activity.details &&
+        this.emoji === activity.emoji)
```

Comment:
Something like this is probably a better idea
```suggestion
        this.emoji?.id === activity.emoji?.id)
```

Reply:
Didn't even consider that, thanks! 👍

Reply:
don't built in emojis not have an id?